src/main/java/com/microsoft/jfr/generation/JfcToJava.java

```diff
@@ -0,0 +1,28 @@
+package com.microsoft.jfr.generation;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This class allows you to convert a .jfc configuration file into a Java class.
+ */
+public class JfcToJava {
+
+    private JfcToJava() {
+    }
+
+    /**
+     * Converts a .jfc configuration file into a Java class that provides a <code>java.util.Map</code> containing a JFR event configuration.
+     * @param jfcPath Path to a .jfc file
+     * @param javaClassDirPath Path directory of the generated Java class
```

Comment:
So this is a .java file, not a .class file, correct? I'm getting a little tripped up on the terminology. Maybe generatedSourcePath?